### PR TITLE
feat: add inbox

### PR DESCRIPTION
**Pull request by TimoGlastra**
https://github.com/animo/paradym-wallet/assets/23165168/460f3b4c-a874-49a1-99c6-ba228dad4afa

### CODE REVIEW

File: packages/app/components/NoContentInbox.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const { back } = useRouter()
+
+  return (
+    <YStack jc="space-between" px="$4" height="95%">
+      <Spacer />
+      <YStack>
+        <YStack jc="center" ai="center" gap="$2">
+          <Heading variant="h2">You're all caught up.</Heading>
+          <Paragraph textAlign="center" secondary>
+            You don't have any notifications at the moment.
```
Comment (Tommylans) [inline packages/app/components/NoContentInbox.tsx:14]: Maybe this a bit smaller than the title?

**Review by Tommylans — APPROVED** [review]
Looks good maybe a small styling improvement but I'm not sure of how it would look. It can also look a bit odd maybe good to ask @janrxyz later for a quick look :smile:.

File: packages/app/components/NoContentInbox.tsx (hunk 1 above)
Comment (TimoGlastra) [reply packages/app/components/NoContentInbox.tsx:14]: I kept it the same as the home screen when you don't have any crednetials yet. Shouild i also change that you think for consistency?

Comment (TimoGlastra) [reply packages/app/components/NoContentInbox.tsx:14]: Merged for now, but let me know, and I'll make a follow up PR

**Pull request merged**